environments/prod/childarrangement-familyinjunction-service-gov-uk.yml

```
@@ -0,0 +1,29 @@
+ns:
+
+mx:
+
+A:
+
+txt:
```

if you add these changes you'll be able to revert your last commit, it's recommended that you add the different record types as otherwise people will get confused when adding an A record or a txt record doesn't work (mx and ns don't matter so much as unlikely to be used here)
```suggestion
ns: []

mx: []

A: []

txt: []
```